metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDBTest.java

```diff
@@ -86,14 +86,15 @@ public void importStudy() throws Exception {
     @Test
     @Category(OracleDbCategory.class)
     public void importAnalysisObject() throws Exception {
-        Analysis analysis = sraObjectImporter.importAnalysis("ERZ000011");
+        Analysis analysis = sraObjectImporter.importAnalysis("ERZ019262");
```

Comment:
Can you please add test case for analysis with empty refSequence accession.

Reply:
Added in https://github.com/EBIvariation/metadata-ws/pull/58/commits/5e089fff7bc217fd3c7eb42c4d4e3c0c805aece9